### DISCUSSION TITLE
Is it normal for links to look like this?

### DISCUSSION THREAD

**Discussion by Je12emy** [nvim-neorg/neorg#293] [General]
Links look like this on my terminal, and I was wondering if this is how they are supposed to look like.

![image](https://user-images.githubusercontent.com/34408108/147947435-ed3ecda9-7b10-45db-8711-c65b2b15a428.png)

**Comment by max397574** [comment]
Yes this is how they're supposed to look like this if you have the default settings.
The problem is that because of the 0-width unicode character which was used to conceal stuff some concealing broke.
Now stuff to be concealed is just replaced with whitespace that's why there is space before the links. The _______ from the URL Link is there because the highlight for the URL has an underline.

you can get the "old" concealing (without whitespace) by settings `config.markup_preset` to `"brave"` (default is `"safe"`) in the concealer setup

**Reply by Je12emy** [reply]
That's just what I needed, thank you!

![image](https://user-images.githubusercontent.com/34408108/147953885-42fe581f-9825-416c-81bf-d59096ee688d.png)

**Comment by danymat** [comment]
Related: https://github.com/nvim-neorg/neorg/issues/280
